### ISSUE TITLE
Sync in sound

### ISSUE THREAD

**Issue by nikhilCad** [Ankit404butfound/Piano#2]
When you start typing keys like crazy, the audio files cut each other. 
You should play all keystroke till their end instead of cutting the audio play in between.

**Comment by nikhilCad** [comment]
https://stackoverflow.com/questions/33574945/how-to-play-multiple-sounds-simultaneously-using-winsound/48456386

As per this, winsound does not seem to support this. But this should be a basic feature for piano

**Comment by nikhilCad** [comment]
Using pygame is best option, if you want to do it. Feel free to ignore.

**Comment by Ankit404butfound** [comment]
Yes, pygame  should work, I haven't tried because I was just testing the code and also the real piano also cuts the sound, I guess, not sure.

**Comment by Ankit404butfound** [comment]
Also, adding a time delay of 0.5 seconds should work too.

**Comment by nikhilCad** [comment]
> Also, adding a time delay of 0.5 seconds should work too.

Not exactly, all it does is prevent users from spamming keys on the keyboard. Multiple sounds are simply not supported in winsound

**Comment by Ankit404butfound** [comment]
Ok, I will better use pygame.
